oracle/blockoracle.go

```diff
@@ -38,42 +44,47 @@ func NewMinerBlockOracle(committeeSize int32, layersPerEpoch uint16, activationD
 		vrfSigner:      vrfSigner,
 		nodeID:         nodeId,
 		proofsEpoch:    ^types.EpochId(0),
+		log:            log,
 	}
 }
 
 func (bo *MinerBlockOracle) BlockEligible(layerID types.LayerID) (types.AtxId, []types.BlockEligibilityProof, error) {
 
 	epochNumber := layerID.GetEpoch(bo.layersPerEpoch)
-
+	bo.log.Info("asked for eligibility for epoch %d (cached: %d)", epochNumber, bo.proofsEpoch)
 	if bo.proofsEpoch != epochNumber {
 		err := bo.calcEligibilityProofs(epochNumber)
 		if err != nil {
-			return types.AtxId{}, nil, err
+			bo.log.Error("failed to calculate eligibility proofs: %v", err)
+			return types.AtxId{common.HexToHash("EEEEEEEE")}, nil, err
```

Comment:
Any other places you use "EEEEE..." ?
Either way, consider using a const

Reply:
for debugging purposes only, it is supposed to be empty